middlewares/middlewares.go

```diff
@@ -104,12 +108,33 @@ func AccessControl() negroni.Handler {
 
 // JwtMiddleware check if actual request have JWT
 func JwtMiddleware(key string, algo string) negroni.Handler {
-	jwtMiddleware := jwtmiddleware.New(jwtmiddleware.Options{
-		ValidationKeyGetter: func(token *jwt.Token) (interface{}, error) {
-			return []byte(key), nil
-		},
-		SigningMethod: jwt.GetSigningMethod(algo),
-	})
+	issuerURL, err := url.Parse("https://127.0.0.1/")
+	if err != nil {
+		log.Fatalf("Failed to parse the issuer url: %v", err)
+	}
+	provider := jwks.NewCachingProvider(issuerURL, 5*time.Minute)
```

Comment:
If we're using the jwks provider we need to make sure that there is one up and running at the issuerURL provided above and that has the openid-configuration under `/.wellk-nown/openid-configuration`. 

Alternatively we could bypass getting the OIDC by passing the `jwks.WithCustomJWKSURI` option to the provider e.g. `jwks.NewCachingProvider(issuerURL, 5*time.Minute, jwks.WithCustomJWKSURI(jwksURI))`. 

In case none of the above work we can always provide a custom `keyFunc` to fetch the data with which the JWT has been signed with.